### DISCUSSION TITLE
More beatmap colours for osu!mania and or customizable colours for osu!mania

### DISCUSSION THREAD

**Discussion by Arcadris** [ppy/osu#17671] [Ideas]
I love the default osu!lazer theme but I am not a huge fan of the colours of the lanes in osu!mania. I am currently trying to create a skin that changes the lane colours but having an option to change the colours in-game would be very helpful. Along with changing the lane colours as part of the beatmap could add more variety to charts. Just 3 colours for note 1, note 2, and special notes would be fine by me though others may want more customization and I can see how more would be helpful for higher key counts.

**Comment by peppy** [comment]
This will be feasible in the future via the skin editor.

**Comment by iSimpFor** [comment]
there could be 2 settings for mania combo colors: regular and manual

regular sets colors for each skin file(note 1, 2, and special) while manual sets colors for each lane
